elb.tf

```diff
@@ -62,10 +61,12 @@ resource "aws_lb_listener" "this" {
 
 resource "aws_lb_target_group" "this" {
   name_prefix = substr(var.name, 0, 6)
-  port        = "80"
-  protocol    = "HTTP"
-  tags        = var.tags
-  vpc_id      = var.vpc_id
+  # Nexus has a bad time if two instances are running at once, so the deregistration delay needs to be short
```

Comment:
How dangerous is it to set the delay to 0 and just terminate the old instance?

Reply:
I'm not sure. I thought about it but I figured at least this way there's no chance Nexus would still have a lock on the running repo by the time cloud-init finished on the new host, but there would be _some_ time for the Java process to shut down naturally-ish.